### DISCUSSION TITLE
Exploring the use of CWatM for the PhD project at the University of Geneva

### DISCUSSION THREAD

**Discussion by igorchernov88** [iiasa/CWatM#126] [Q&A] [answered]
Dear CWatM community,
We are two PhD students working on a research project "Water-Soil-Food-Climate Nexus of Greater Geneva area" at the University of Geneva. We are applying [MuSIASEM (Multi-scale Integrated Analysis of Societal and Ecosystem Metabolism)](https://en.wikipedia.org/wiki/MuSIASEM) to the Greater Geneva region in Switzerland to assess local water uses and their environmental sustainability. We would like to integrate this analysis with a hydrological model. For this, we were exploring to use [Soil & Water Assessment Tool (SWAT)](https://swat.tamu.edu), but recently learnt about CWatM. In this regard, we would be very grateful to get some information on the following questions:
- What is the smallest spatial- and temporal resolution that can be used in CWatM when modeling a specific local watershed for which high-resolution input data is available?
- Has someone already compared SWAT and CWatM models? If yes, what are the main differences and specialities?

Thank you very much in advance,
Igor & Alex

**Comment by dof1985** [comment, marked as answer]
Dear Igor & Alex,

Thanks for reaching out. I hope you find CWatM useful for your purpose. Regarding your questions:

- CWatM can be applied at a variety of spatial resolutions ranging between 0.5 degrees to 30'' (~1 km; the highest), and runs at a daily time step. Having said that - the landcover is represented at a higher resolution as the fraction of a grid cell, and some processes (soil, routing) are simulated at a sub-daily timestep. 1 km resolution applications includes Bhima basin (India), EBRO basin (Spain), and even a small basin in Israel (Ayalon, ~840 km2).
- I am not aware of a comparison. The two models take slightly different approaches. One major difference is in how the two models handle crops. Currently, CWatM can simulate crops' water requirements and yield loss associated with water scarcity and a coupling with the Global Agroecological Zones is ongoing (see also here: [https://github.com/iiasa/CWatM/discussions/83]; SWAT had integrated the EPIC crop growth model, which is a complete crop model. I also propose to read the SWAT manual (SWAT2009 Theory) here: [https://swat.tamu.edu/docs/](https://swat.tamu.edu/docs/).

Hope it helps
Dor

**Reply by igorchernov88** [reply]
Dear Dor,
Thank you very much for your response and all the information. Is the low maximum spatial resolution (30'' or ~ 1 km) due to the fact that some of the input data have a spatial resolution of 1 km2? Are these climate data? Is it possible to run the model with higher resolution input data (e.g., 0.01 km2)?
Thank you again. Have a nice day, Igor

**Reply by dof1985** [reply]
Dear Igor.

I am glad that you have found my initial response helpful, and I am sorry for not clarifying it beforehand. The reason for the 1km limit, is the temporal resolution - namely, we find the daily time step unsuitable for modelling hydrological process in higher (than 1 km) spatial resolutions, as the exact timing of the hydrological response (e.g., runoff formation) becomes much more important. 
I am not convinced that you actually need higher resolution for the application you mentioned, but you would know the desired outcomes better.

Hope this answer makes it clearer
Dor

**Reply by igorchernov88** [reply]
Dear Dor,
Thank you very much for your prompt response and your availability. It is very helpful! We will continue to investigate CWatM and would be happy to connect with you and your team once we have better understanding of the tool.
Thanks again and have a wonderful day,
Igor
